implement-cowsay/main.py

```diff
@@ -0,0 +1,20 @@
+import cowsay
+import argparse
+import sys
+
+
+parser = argparse.ArgumentParser(description="displays cowsay charactor with given arguments")
+parser.add_argument('animal', type=str, help= 'please enter a valid animal')
+parser.add_argument('message', type=str, help= 'please enter a message')
+arg = parser.parse_args()
+
+valid_animals = cowsay.char_names
```

Comment:
There is a way to validate the lsit of animals though argparse directly (check out `choices` argument).